packages/emoji-mart/src/components/Picker/PickerStyles.scss

```diff
@@ -46,6 +46,10 @@
 
   --sidebar-width: 16px;
 
+  --preview-placeholder-size: 21px;
```

Comment:
What I realize here @EtienneLem is that we should probably use an internal variable name and check if a global variable is present or if the value should default to 21px. Otherwise, there's no way to customize the size.

Reply:
I've created this small PR to address the override of the CSS variable: https://github.com/missive/emoji-mart/pull/779

Reply:
Should be working in **v5.5.1**